### ISSUE TITLE
`:pyro|commit` misses `ford` cache when inserting changed files

### ISSUE THREAD

**Issue by tadad** [uqbar-dao/uqbar-core#359]
A sample of some verbose output (where it says `ford:cache [...]: creating` tends to hang the longest)
```
ford: cache [%file path=/sur/zig/subscriber/hoon]: creating
ford: cache [%file path=/sur/zig/subscriber/hoon] for [%vale path=/sur/zig/subscriber/hoon lobe=0v1k.21485.jq1hp.6ji4j.hmcs4.mr7lb.nue3t.a1062.6jmem.8nl00.b2pko]: bumping to ref 1
ford: spilt [%file path=/sur/zig/subscriber/hoon]
ford: cache [%file path=/app/subscriber/hoon]: creating
ford: cache [%file path=/app/subscriber/hoon] for [%file path=/lib/dbug/hoon]: bumping to ref 16
ford: cache [%file path=/app/subscriber/hoon] for [%file path=/lib/default-agent/hoon]: bumping to ref 19
ford: cache [%file path=/app/subscriber/hoon] for [%vale path=/app/subscriber/hoon lobe=0ve.lv11d.ppmst.c0tak.1nhv4.ruisg.c58hd.5pgr3.9h5e7.q72gh.01i7b]: bumping to ref 1
ford: cache [%file path=/app/subscriber/hoon] for [%file path=/sur/zig/subscriber/hoon]: bumping to ref 1
ford: spilt [%file path=/app/subscriber/hoon]
```
generate this by running : `:pyro|dojo ~nec "|pass [%c %stir %verb 5]"` and then using `:pyro|commit`.

**Comment by hosted-fornet** [comment]
Is this issue resolved by #361?

**Comment by tadad** [comment]
yes
